AirCasting/Settings/MyAccountViewSignOut.swift

```diff
@@ -94,4 +101,17 @@ private extension MyAccountViewSignOut {
         .padding(.bottom, 20)
         .padding()
     }
+    
+    private func logoutUser() {
+        userState.currentState = .loggingOut
```

Comment:
I think we can move setting userState from here to the `LogoutController` too. This kind of logic should not be present in Views